overlord/devicestate/devicemgr.go

```diff
@@ -371,6 +374,48 @@ func (m *DeviceManager) ensureBootOk() error {
 	return nil
 }
 
+func markSeededInConfig(st *state.State) error {
+	var seedDone bool
+	tr := config.NewTransaction(st)
+	if err := tr.Get("core", "seed.done", &seedDone); err != nil && !config.IsNoOption(err) {
```

Comment:
why seed.done instead of seeded?

Reply:
also that construction to be more readable would need to be "seeding.done"

Reply:
also looking at "man systemd.special" most targets are based on names, not adjectives:

so snapd.seeding or snapd.seed